elastic/integrations
# Removing detection rules from the `problemchild` package

## What does this PR do?
This PR moves detection rules associated with DGA (dga) out of the integration package. 

## Checklist

## How to test this PR locally
- Spin up a local Elasticsearch cluster
- Verify that the package version shown is 2.0.0, Subscription is `platinum` and License is `Elastic-2.0`
- Install the Living off the Land Detection assets
- Verify that no detection rules with the `Living off the Land` tag show up on the Rules Management page under Security
- Verify that `-embedding` does not appear on the blocklist processor in the inference pipeline

## Related Issues
Not linking since these are internal issues

## Screenshots
-  License and subscription info is as expected
<img width="1280" alt="Screen Shot 2023-09-21 at 10 22 09 AM" src="https://github.com/elastic/integrations/assets/30438249/55bd192b-baa5-4d8e-8152-6c3161d4c2d7">

-  No rules appear upon installation of package assets
<img width="1513" alt="Screen Shot 2023-09-21 at 10 22 44 AM" src="https://github.com/elastic/integrations/assets/30438249/bfa1f56a-6c2a-46d1-810f-e1cc35644bba">

-  `-embedding` is not part of the blocklist in the inference processor
<img width="533" alt="Screen Shot 2023-09-21 at 10 23 56 AM" src="https://github.com/elastic/integrations/assets/30438249/7738ddfa-16c1-43e1-8e69-d08b577b788a">
